### DISCUSSION TITLE
Feature request: bank search for quest items

### DISCUSSION THREAD

**Discussion by Raisoshi** [Zoinkwiz/quest-helper#685] [Ideas]
Similar to when you search "clue" with the clue plugin, where it shows you the required items if you have an active emote step, could we possibly search "quest" or even better add a buttom to sort the bank and show the required items listed in the quest helper side bar?

**Comment by Atroxide** [comment]
![image](https://user-images.githubusercontent.com/546362/153992655-59c9451d-e59b-450c-a95a-8e488698c655.png)

Click the quest icon in top right panel of the bank screen.
